configuration/debian/tedge_agent/postrm

```diff
@@ -42,4 +43,22 @@ case "$1" in
     ;;
 esac
 
-#DEBHELPER#
+# Automatically added by cargo-deb
```

Comment:
This comment and all others related to cargo-deb are a bit confusing since we don't use cargo-deb any more for the agent.

Reply:
Ok, removed that.